iroh-relay/src/protos/common.rs

```diff
@@ -3,55 +3,65 @@
 //! Hosts the [`FrameType`] enum to make sure we're not accidentally reusing frame type
 //! integers for different frames.
 
-use bytes::{BufMut, Bytes};
+use bytes::{Buf, BufMut};
+use nested_enum_utils::common_fields;
 use quinn_proto::{coding::Codec, VarInt};
+use snafu::{Backtrace, OptionExt, Snafu};
 
 /// Possible frame types during handshaking
 #[repr(u32)]
-#[derive(Copy, Clone, PartialEq, Eq, Debug, num_enum::IntoPrimitive, num_enum::FromPrimitive)]
+#[derive(
+    Copy, Clone, PartialEq, Eq, Debug, num_enum::IntoPrimitive, num_enum::TryFromPrimitive,
+)]
 // needs to be pub due to being exposed in error types
 pub enum FrameType {
     /// The server frame type for the challenge response
-    ServerChallenge = 2,
+    ServerChallenge = 1,
```

Comment:
Why not start from 0?  Though it's not the end of the world, we can always use that frame later when we want to.